examples/docker/echo_app.py

```diff
@@ -0,0 +1,52 @@
+# Example using SkyPilot python API to run the echo app in a container.
+#
+# The echo example ingests a file, prints the contents and writes it back out.
+# In this YAML, the output is mapped to a Sky Storage object, which writes to a
+# cloud bucket.
+#
+# Usage:
+#   python echo_app.py
+
+import random
+import sky
+import string
+
+with sky.Dag() as dag:
+    # The setup command to build the container image
+    setup = 'docker build -t echo:v0 /echo_app'
+
+    # The command to run - runs the container and mounts volumes
+    run = 'docker run --rm ' \
```

Comment:
nit: Use parentheses instead of backslash?

Reply:
good catch, thanks! I really need to configure my ide to use parenthesis haha